tests/test_scene_optimizer.py

```diff
@@ -87,60 +62,52 @@ def test_find_largest_connected_component(self):
         (
             computed_relative_rotations,
             computed_relative_unit_translations,
-        ) = select_largest_connected_component(
-            input_relative_rotations, input_relative_unit_translations
-        )
+        ) = select_largest_connected_component(input_relative_rotations, input_relative_unit_translations)
```

Comment:
Are you using 120 line width?

Reply:
yup

Reply:
I generally am thinking these days 120 length makes for more readability in the single column format in the github.com UI.

In 2-column review, 80 chars can be a bit easier, but widening the browser just for review works as a fix for me.